src/sidecar/connections.test.ts

```diff
@@ -133,6 +133,40 @@ describe("sidecar/connections.ts", () => {
     setContextValue(ContextValues.ccloudConnectionAvailable, true);
     assert.strictEqual(hasCCloudAuthSession(), true);
   });
+});
+
+describe("sidecar/connections.ts waitForConnectionToBeUsable() tests", () => {
```

Comment:
This diff is nasty to read. Easier to read this and subsequent section of file verbatim, in that the existing test suite needed to be split in half.